src/VarshaT/Program3.java

```diff
@@ -0,0 +1,38 @@
+/*
+ 3)	WAP to find frequency of user defined character in user defined String
+a.	i.e. User String :- �electrical engineering�
+b.	char value :- e
+c.	count of e :- 5
+
+ */
+package VarshaT;
+import java.util.Scanner;
+public class Program3 {
+
+		void freq(String str, char c)
+		{
+			String s[]=str.split("e");
```

Comment:
Hi @VarshaTAug19 
Logic is not working as expected please check and debug code...